meteor_packages/mats-common/templates/params/scatter_axis.html

```diff
@@ -1,40 +1,95 @@
 <template name="scatter2d">
-    <div class="well well-sm pull-left" id={{plotType}}  style="display:{{displayScatter2d}};">
-        <div style="font-size:large; font-weight: bold;">{{title}}</div>
+  <div
+    class="well well-sm pull-left"
+    id="{{plotType}}"
+    style="display:{{displayScatter2d}};"
+  >
+    <div style="font-size: large; font-weight: bold">{{title}}</div>
+    <div>
+      <span class="btn-success" style="font-size: large;display:{{creating}};"
+        >{{modeText}}</span
+      >
+      <span class="btn-info" style="font-size: large;display:{{editing}};"
+        >{{modeText}}</span
+      >
+    </div>
+    {{#each scatter2dParams}}
+    <div class="well well-sm pull-left">
+      {{#if isNumberSpinner this}}
+      <input
+        class="data-input input-sm"
+        id="{{idParam}}"
+        type="number"
+        value="{{default}}"
+        name="{{name this}}"
+        min="{{min}}"
+        max="{{max}}"
+        step="{{step}}"
+      />
+      <label for="{{idParam}}">{{labelParam}}</label>
+      {{else}}
+      <h4>
+        {{name this}}
+        <small
+          >{{#if hasHelp}}
+          <button
+            style="background-color: #bbffbb"
+            type="button"
+            class="btn btn-default btn-sm help"
+          >
+            <i class="fa-solid fa-question"></i>
+          </button>
+          {{/if}}
+        </small>
+      </h4>
+      {{#each scatter2dOptions}}
+      <input
+        class="data-input {{className ..}}"
+        type="{{type"
+        ..}}
+        name="{{name ..}}"
+        id="{{idOption ..}}"
+        value="{{this}}"
+        {{isDefault
+        ..}}
+      />
```

Comment:
@randytpierce - is this valid handlebar syntax? Prettier is complaining that the path expression isn't valid and thus mangling it. I suspect as this is written we're pulling e.g. `type` from the current context and not from the parent context. It seems like these could be re-written as `{{name}}` instead of `{{name ..}}`. Does that sound right to you?